src/main/resources/com/mathworks/ci/MatlabInstallationAxis/help.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+        Use this list to add MATLAB versions as axes to Matrix configuration.
+        The plugin prepends your specified MATLAB axis value to the PATH environment variable and
+        uses it to perform the matrix build. <br>
+        The list includes the names of all MATLAB versions that have been registered as a tool in Jenkins.
+    </p>
+    <p><br>
+        <b>Note:</b> In multi-configuration projects, the value specified by <b>Use MATLAB version</b> takes precedence over the values specified by the <b>MATLAB</b> axis.
```

Comment:
I am thinking maybe it's better to say "the values ... take precedence" instead of "the value ... takes precedence". This is because "Use MATLAB version" most probably uses a user-defined axis name, which represents multiple values defined by the axis. Does this make sense?

Reply:
Done.